Shared/Store/BaseDataStore.swift

```diff
@@ -229,21 +231,29 @@ extension BaseDataStore {
             else { return }
 
         if (networkStore.isConnectedToNetwork) {
-            self.syncSubject.onNext(SyncState.Syncing)
+            self.syncSubject.accept(SyncState.Syncing)
         } else {
-            self.syncSubject.onNext(SyncState.Synced)
+            self.syncSubject.accept(SyncState.Synced)
             return
         }
 
         queue.async {
+            self.queue.asyncAfter(deadline: .now() + 20, execute: {
```

Comment:
this queues for 20 seconds from `.now()`, correct?  I ask because I am not super-familiar with the interface, and most other systems seem to use milliseconds.

Also, is it worth making `20` a constant?

Reply:
yes, it queues for 20 seconds from now. Constant is a good idea!